### ISSUE TITLE
Strange output with red noise point

### ISSUE THREAD

**Issue by AlexCHENSiyu** [ivanstepanovftw/artistic-style-transfer-for-videos#3]
![styled_38_0](https://github.com/ivanstepanovftw/artistic-style-transfer-for-videos/assets/80956056/89d444e0-676a-4f5c-a0af-0c1012ceddcf)
I can get this good output yesterday. 
but today when I ran it again, I get this strange output:
![styled_39_1](https://github.com/ivanstepanovftw/artistic-style-transfer-for-videos/assets/80956056/7abe31cf-d403-4e7a-8ffe-d820b983ca5a)

Have you ever encountered this problem before? Or you can give me some suggestions?

**Comment by AlexCHENSiyu** [comment]
ok, probably I find the reason. BTW , this part should be written like this:
![image](https://github.com/ivanstepanovftw/artistic-style-transfer-for-videos/assets/80956056/ac604189-9df5-47d3-a036-1ecd3e95877e)
add ['arr_0'] to it. otherwise cannot run.

**Comment by ivanstepanovftw** [comment]
I had this when I did not clamp styled image: https://github.com/ivanstepanovftw/artistic-style-transfer-for-videos/blob/b42b79d17fd1b72917b55100182c6f5f3d097d92/main_video_join.py#L57
Similar as it is performed in the closure: https://github.com/ivanstepanovftw/artistic-style-transfer-for-videos/blob/b42b79d17fd1b72917b55100182c6f5f3d097d92/main_video.py#L342
I actually thinking about adding loss for out of bounds pixels, so the optimizer would not be focused on these pixels.

**Comment by AlexCHENSiyu** [comment]
yes, right after I posted the issue, I found this mistake of mine. 
I read the paper carefully and I realize that long-term temporal consistency requires heavy-duty computation. It might be hard for me to achieve that.

**Comment by AlexCHENSiyu** [comment]
BTW, do you know any comparative standard to measure the quality of style-transferred images between different pre-train model?

**Comment by AlexCHENSiyu** [comment]
and any reason why you choose efficientnet-b0 as the best model

**Comment by ivanstepanovftw** [comment]
> BTW, do you know any comparative standard to measure the quality of style-transferred images between different pre-train model?

Unfortunately, no.

> and any reason why you choose efficientnet-b0 as the best model

I am not sure if it is best for style transfer. EfficientNet chosen as it efficient, since I do not have local GPU.

Also, in the paper "CNN Filter DB: An Empirical Investigation of Trained Convolutional Filters" ([arxiv:2203.15331v2](https://arxiv.org/abs/2203.15331v2)), they say that "learned filters do not significantly differ across models trained for various tasks, except for extreme outliers such as GAN-Discriminators."
